src/base/map_info.ixx

```diff
@@ -390,7 +407,29 @@ export class MapInfo {
 		writer.write(playable_width);
 		writer.write(playable_height);
 
-		const int flags = hide_minimap_preview * 0x0001 | modif_ally_priorities * 0x0002 | melee_map * 0x0004 | unknown * 0x0008 | masked_area_partially_visible * 0x0010 | fixed_player_settings * 0x0020 | custom_forces * 0x0040 | custom_techtree * 0x0080 | custom_abilities * 0x0100 | custom_upgrades * 0x0200 | unknown2 * 0x0400 | cliff_shore_waves * 0x0800 | rolling_shore_waves * 0x1000 | unknown3 * 0x2000 | unknown4 * 0x4000 | item_classification * 0x8000 | water_tinting * 0x10000 | accurate_probability_for_calculations * 0x20000 | custom_ability_skins * 0x40000;
+		const int flags = hide_minimap_preview * 0x0001 
+						| modif_ally_priorities * 0x0002
+						| melee_map * 0x0004
+						| unknown * 0x0008
+						| masked_area_partially_visible * 0x0010
+						| fixed_player_settings * 0x0020
+						| custom_forces * 0x0040
+						| custom_techtree * 0x0080
+						| custom_abilities * 0x0100
+						| custom_upgrades * 0x0200
+						| unknown2 * 0x0400
+						| cliff_shore_waves * 0x0800
+						| rolling_shore_waves * 0x1000
+						| unknown3 * 0x2000
+						| unknown4 * 0x4000
```

Comment:
> IF( TFT ) 	
0x002000 	Use terrain fog
0x004000 	Expansion required (TFT)

Reply:
I won't do these changes in this PR as it's not the topic, but feel free to make a separate one for this, thanks!